docs/getting-started-rack.asciidoc

```diff
@@ -1,9 +1,26 @@
 [[getting-started-rack]]
 == Getting started with Rack
 
+Add the gem to your `Gemfile`:
+
+[source,ruby]
+----
+gem 'elastic-apm'
+----
+
+Create a file `config/elastic_apm.yml`:
+
+[source,yaml]
+----
+server_url: http://localhost:8100
+secret_token: ''
+----
+
+Include the middleware, start (and stop) Elastic APM when booting your app:
+
 [source,ruby]
 ----
-# config.ru
+# Example config.ru
```

Comment:
Could you also add a pure Rack example, as this section is generally called `Getting started with Rack`..